app/shared/language/language-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { LanguageSchema } from 'shared/types/commonTypes';
+import { LanguageCode, availableLanguages, OTHER_LANGUAGE_SCHEMA } from './available-languages';
+
+class LanguageUtils {
+  static getLanguagesByCode(code: LanguageCode = 'elastic'): LanguageSchema[] {
+    return availableLanguages.filter(item => Boolean(item[code]));
+  }
+
+  private static uniqueValues(array: string[]) {
```

Comment:
Maybe just for readability move the private method to the top of the class definition?